File: PenultWord.py
import sys
test_cases = open("/home/carlos/Documents/CodeEval/TestFiles/PentulWord",'r')
for test in test_cases:
	words = test.split(" ")
	for n in range(len(words)-1):
		if words[n] > words [n+1]:
			tmp = words[n]
			words[n] = words[n+1]
			words[n+1] = tmp
		print words[len(words)-1]
test_cases.close()

File: PenultWord.c
#include <stdio.h>
int main(int argc, const char * argv[]) {
    FILE *file = fopen("/home/carlos/Documents/CodeEval/TestFiles/PenultWord", "r");
    char line[1];
    while (fgets(line, 1, file)) {
        printf("%s",line);
    }
    return 0;
} 
